packages/app/providers/web3-provider.tsx

```diff
@@ -63,6 +66,28 @@ export function Web3Provider({
     }
   }, [connector]);
 
+  // (Native only) initializes wallet mobile sdk provider
+  useEffect(() => {
+    (async function setWeb3Provider() {
+      if (Platform.OS !== "web" && mobileSDK.connected && mobileSDK.address) {
+        const MobileSDKProvider = (
+          await import(
+            "@coinbase/wallet-mobile-sdk/build/WalletMobileSDKEVMProvider"
+          )
```

Comment:
Why are you dynamically importing here? await import syntax looks kinda ugly IMHO.

Reply:
Was following the convention used in this file. Not sure if there's a reason why it was done this way here. https://github.com/vishnumad/showtime-frontend/blob/staging/packages/app/providers/web3-provider.tsx#L47-L67

Reply:
We're using dynamic imports to lazy load packages, looks ugly I agree, but need to take care of bundle size on web 😄